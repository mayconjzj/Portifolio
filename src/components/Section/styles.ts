import tw from 'tailwind-styled-components';

export const Container = tw.section`
  flex
  flex-col
  gap-5
  px-[10%]
  py-20
  border-t-2
  border-t-[#111]
  min-h-[60vh]
`;
