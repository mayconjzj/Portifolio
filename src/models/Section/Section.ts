export type SectionProps = {
  children: React.ReactNode;
  id: string;
};
