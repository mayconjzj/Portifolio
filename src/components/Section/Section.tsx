import { SectionProps } from '@/models';

import * as S from './styles';

export const Section = ({ children, id }: SectionProps) => {
  return (
    <S.Container id={id} className="element">
      {children}
    </S.Container>
  );
};
