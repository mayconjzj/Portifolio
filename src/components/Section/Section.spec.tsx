import { render } from '@testing-library/react';

import { Section } from './Section';

describe('MainNav', () => {
  it('should render the mainnav', () => {
    const { container } = render(<Section>Next test</Section>);
    expect(container).toMatchSnapshot();
  });
});
