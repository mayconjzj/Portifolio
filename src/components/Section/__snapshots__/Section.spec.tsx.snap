// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainNav should render the mainnav 1`] = `
<div>
  <section
    class="flex flex-col gap-5 px-[10%] py-20 border-t-2 border-t-[#111] min-h-[60vh]"
  >
    Next test
  </section>
</div>
`;
